src/gui/Previewers.hpp

```diff
@@ -39,6 +39,8 @@ class Previewers {
   enum class FileType { Unkown, Image, Video, Audio, Text };
 
  public:
+  Previewers() = default;
```

Comment:
These is not needed since we can allow the compiler to create a default instead.